design/high-level-reactivity.md

```diff
@@ -234,12 +296,20 @@ The new system is designed to avoid this issue, and the implementation will be b
 
 After you analyze the two paths above, there should be a clear path to optimization through careful manual invalidations and disabling reactivity that will let you change a minimum of app code to "twist the knobs" on performance.
 
+// Oh Yeah! I'm seeing huge set of ideas. 
+// Let's implement this. So, we can build a great APM and make money :P
+
 ## Implementation plan
 
 Here's a diagram of all of the pieces we think will need to be built to have a complete system:
 
 ![Block diagram](block-diagram.png)
 
+// What does FlowRouter has to do with this. Do we need to do any changes?
+
 It's a lot of stuff, but a lot of it already exists thanks to the Relay project, and some of the things can be contributed by the community once the structure is clearer, for example some of the database drivers.
 
+// Are we using Relay's connection API which is optimized for pagination. That's place where it adds the complexity to Relay and it's GraphQL part.
+// With our design, I don't think we need it.
```

Comment:
It's not about its bad. It's good. 
But writing schemes with connection is kind hard.

That's something optional in Relay I think. Anyway, if you can up with a simple way to write schemas then there is no issue at all.